src/StoreApi/docs/products.md

```diff
@@ -35,34 +36,36 @@ GET /products?return_attribute_counts=pa_size,pa_color
+| `exclude`                     | array   |    no    | Ensure result set excludes specific IDs.                                                                                                                                   |
+| `include`                     | array   |    no    | Limit result set to specific ids.                                                                                                                                          |
+| `offset`                      | integer |    no    | Offset the result set by a specific number of items.                                                                                                                       |
+| `order`                       | string  |    no    | Order sort attribute ascending or descending. Allowed values: `asc`, `desc`                                                                                                |
+| `orderby`                     | string  |    no    | Sort collection by object attribute. Allowed values : `date`, `modified`, `id`, `include`, `title`, `slug`, `price`, `popularity`, `rating`, `menu_order`, `comment_count` |
+| `parent`                      | array   |    no    | Limit result set to those of particular parent IDs.                                                                                                                        |
+| `parent_exclude`              | array   |    no    | Limit result set to all items except those of a particular parent ID.                                                                                                      |
+| `type`                        | string  |    no    | Limit result set to products assigned a specific type.                                                                                                                     |
+| `sku`                         | string  |    no    | Limit result set to products with specific SKU(s). Use commas to separate.                                                                                                 |
+| `featured`                    | boolean |    no    | Limit result set to featured products.                                                                                                                                     |
+| `category`                    | string  |    no    | Limit result set to products assigned a specific category ID.                                                                                                              |
+| `category_operator`           | string  |    no    | Operator to compare product category terms. Allowed values: `in`, `not_in`, `and`                                                                                          |
+| `[product-taxonomy]`          | string  |    no    | Limit result set to products assigned to the term ID of that custom product taxonomy. `[product-taxonomy]` should be the key of the custom product taxonomy registered.    |
```

Comment:
Does this need to be updated so it includes the `tax_` prefix?

Reply:
Yes, I just waited for validation of the logic to be used. I updated here [9d6b1a7](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/6152/commits/9d6b1a744a2c06365c67746c8815121dc0a9fc7a)